pom.xml

```diff
@@ -85,6 +85,7 @@
         <apache.rat.version>0.16.1</apache.rat.version>
         <google.java.format.version>1.8</google.java.format.version>
         <delta.standalone.version>0.5.0</delta.standalone.version>
+        <delta.hive.version>3.0.0</delta.hive.version>
```

Comment:
2.0.2 delta hive is not there in maven: https://repo1.maven.org/maven2/io/delta/delta-hive_2.12/

<img width="697" alt="image" src="https://github.com/user-attachments/assets/b52ddf6a-36b7-4250-aef6-960d1953b20c" />